package.json

```diff
@@ -21,7 +21,7 @@
     "integ:centerpoint": "python3 lib/osml-model-runner-test/bin/process_image.py --image small --model centerpoint",
     "integ:flood": "python3 lib/osml-model-runner-test/bin/process_image.py --image large --model flood",
     "integ:meta": "python3 lib/osml-model-runner-test/bin/process_image.py --image meta --model centerpoint",
-    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"test_type\": \"integ\", \"source_image_key\": \"small.tif\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
+    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"image_uri\": \"s3://osml-test-images-409719124294/small.tif\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
```

Comment:
Since this is changing from a key to a full URI we need to make this an argument instead of hard coding a bucket.

Reply:
Maybe something like this (also adds the required `cli-binary-format` parameter)?
`    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --cli-binary-format raw-in-base64-out --payload \"{\\\"image_uri\\\": \\\"$TS_INTEG_IMAGE_URI\\\"}\" --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
`

Reply:
I'll fix this when I add integ testing to the guidance hooks